pythonforandroid/bootstraps/service_only/build/templates/Service.tmpl.java

```diff
@@ -22,15 +22,10 @@ public int startType() {
     }
     {% endif %}
 
-    {% if foreground %}
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    public boolean getStartForeground() {
-        return true;
+    @override
```

Comment:
Should be `@Override` with uppercase `O` it seems

Reply:
good catch :+1: